astroquery/esa/xmm_newton/core.py

```diff
@@ -110,7 +110,7 @@ def download_data(self, observation_id, *, filename=None, verbose=False,
         if verbose:
             log.info(link)
 
-        response = self._request('GET', link, save=False, cache=True)
+        response = self._request('HEAD', link, save=False, cache=False)
```

Comment:
Shouldn we add `cache` as a selectible kwarg for `download_data`?

Reply:
...yes.  But we don't want to cache the `HEAD` retrieval; that's the check to make sure that the file size is right

Reply:
oops, sure. Sorry for the noise.

Reply:
no, you were actually right - we do a different HEAD request that can't be cached later; this one _can_ be cached.